.vscode/settings.json

```diff
@@ -7,5 +7,8 @@
     "out": true // set this to false to include "out" folder in search results
   },
   "editor.formatOnSave": true,
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "files.watcherExclude": {
```

Comment:
Wasn't this added by metals by msitake?

Reply:
i forgot to mention it. Running extension tests has to be done from vs code insiders and there musn't be any normal vscode instance opened. So when your workflow looks like:
- implement feature in vscode
- write some extension tests in vscode
- kill all vscode instances, open vscode insiders
- run extension tests from insiders
- e.g. code with `metals-vscode` opens, Metals are modifying settings json

I was tired of removing it each time before commit, so I commited it once and for all.